formatTest/unit_tests/expected_output/basicStructures.re

```diff
@@ -1,5 +1,6 @@
 /* Copyright (c) 2015-present, Facebook, Inc. All rights reserved. */
-let run () => TestUtils.printSection "Basic Structures";
+let run () =>
```

Comment:
This seems right because there was no breaking opportunity before, so this diff also fixed something that wasn't working as intended.